apps/prairielearn/src/lib/score-upload.ts

```diff
@@ -1,34 +1,46 @@
-// @ts-check
-const _ = require('lodash');
-const streamifier = require('streamifier');
-const csvtojson = require('csvtojson');
-const sqldb = require('@prairielearn/postgres');
+import * as _ from 'lodash';
+import * as streamifier from 'streamifier';
+import csvtojson = require('csvtojson');
+import * as sqldb from '@prairielearn/postgres';
+import { z } from 'zod';
 
-const { createServerJob } = require('./server-jobs');
-const manualGrading = require('./manualGrading');
+import { createServerJob } from './server-jobs';
+import * as manualGrading from './manualGrading';
+import { IdSchema } from './db-types';
 
 const sql = sqldb.loadSqlEquiv(__filename);
 
+const assessment_info_schema = z.object({
+  assessment_label: z.string(),
+  course_instance_id: IdSchema,
+  course_id: IdSchema,
+});
+
 /**
  * Update question instance scores from a CSV file.
  *
- * @param {string} assessment_id - The assessment to update.
- * @param {{ originalname: string, size: number, buffer: Buffer } | null | undefined} csvFile - An object with keys {originalname, size, buffer}.
- * @param {string} user_id - The current user performing the update.
- * @param {string} authn_user_id - The current authenticated user.
+ * @param assessment_id - The assessment to update.
+ * @param csvFile - An object with keys {originalname, size, buffer}.
+ * @param user_id - The current user performing the update.
+ * @param authn_user_id - The current authenticated user.
+ *
+ * @returns The ID of the job sequence
  */
-export async function uploadInstanceQuestionScores(assessment_id, csvFile, user_id, authn_user_id) {
+export async function uploadInstanceQuestionScores(
+  assessment_id: string,
+  csvFile: { originalname: string; size: number; buffer: Buffer } | null | undefined,
```

Comment:
Consider using `Express.Multer.File` instead of a bespoke type here.

Reply:
I was unaware of this type, I only copied the JSDoc type from the original code. Done in 5336364694739d475eefcc6ddfd418eb1d5e5d30.